ci-operator/jobs/infra-image-mirroring.yaml

```diff
@@ -95,6 +95,54 @@ periodics:
     - configMap:
         name: image-mirror-mappings
       name: config
+- agent: kubernetes
+  cluster: app.ci
+  cron: '@hourly'
+  decorate: true
+  labels:
+    ci.openshift.io/area: openshift-azure
+    ci.openshift.io/role: image-mirroring
+  name: periodic-image-mirroring-openshift-azure
+  spec:
+    containers:
+    - args:
+      - -c
+      - |
+        set -o errexit
+        failures=0
+        for mapping in /etc/imagemirror/mapping_open_cluster_management*; do
+          echo "Running: oc image mirror -f=$mapping --skip-multiple-scopes"
+          if ! oc image mirror -f="$mapping" --skip-multiple-scopes; then
```

Comment:
We need oc-cli here have access to the source images.
Do you mind open the image on app.ci to the public?
This is an example: https://github.com/openshift/release/blob/309118783cd6dc0422ddc8a88d2c786a6ce60105/clusters/app.ci/ocp-kni/admin_ocp-kni_list.yaml#L8

Otherwise, we have to oc-login or provide a pull token in some way.